deploy/katello.spec

```diff
@@ -85,6 +92,7 @@ install -Dp -m0644 %{confdir}/service-wait.sysconfig %{buildroot}%{_sysconfdir}/
 install -Dp -m0644 %{confdir}/%{name}.logrotate %{buildroot}%{_sysconfdir}/logrotate.d/%{name}
 
 install -p -m0644 etc/service-list %{buildroot}%{_sysconfdir}/%{name}/
+mkdir -p %{buildroot}%{_sysconfdir}/%{name}/services.d
```

Comment:
Does this not also need to be declared in the files sections so that it gets laid down?